docs/source/visualization/kepler.rst

```diff
@@ -0,0 +1,236 @@
+.. _kepler-guide:
+
+Kepler.gl Geographic Visualization
+====================================
+
+PyGraphistry integrates with Kepler.gl to enable powerful geographic visualizations of graph data with latitude/longitude coordinates. This feature allows you to overlay nodes and edges on interactive maps with customizable layers, colors, and effects.
+
+Overview
+--------
+
+The Kepler integration provides:
+
+- **Geographic bindings**: Specify lat/lon columns with ``point_longitude`` and ``point_latitude``
+- **Encoding API**: Configure Kepler datasets, layers, and complete visualizations
+- **Layout support**: Convert coordinates to Mercator projection with ``mercator_layout()``
+- **Type-safe configuration**: Use ``KeplerDataset``, ``KeplerLayer``, and ``KeplerEncoding`` classes
+- **GPU/CPU support**: Process geographic data efficiently on either platform
+
+Quick Start
+-----------
+
+Basic geographic visualization:
+
+.. code-block:: python
+
+    import graphistry
+    import pandas as pd
+
+    # Sample data with coordinates
+    nodes_df = pd.DataFrame({
+        'nodeId': ['NYC', 'LA', 'Chicago'],
+        'lat': [40.7128, 34.0522, 41.8781],
+        'lon': [-74.0060, -118.2437, -87.6298]
+    })
+
+    edges_df = pd.DataFrame({
+        'src': ['NYC', 'NYC'],
+        'dst': ['LA', 'Chicago']
+    })
+
+    # Create geographic visualization
+    g = graphistry.edges(edges_df, 'src', 'dst').nodes(nodes_df, 'nodeId')
+    g = g.bind(point_longitude='lon', point_latitude='lat')
+    g.plot()
+
+Examples
+--------
+
+Point Layer Visualization
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Visualize nodes as points on a map:
+
+.. code-block:: python
```

Comment:
What data does this match up to? The above example has `nodeId` but no col `Cities`... this example feels incomplete->confusing...